### PR TITLE
Moved candidate and party CRUD to event

### PR DESCRIPTION
**Pull request by a1719756**
Moves all the party and candidate creation to be inside a voting event to make creation of data easier.

### CODE REVIEW

**Review by benedict-soh — APPROVED** [review]
Looks good. But probably wait for @LittenJV to review it before merging anyway.

**Review by LittenJV — APPROVED** [review]
looks good

**Pull request merged**
